schema/gen/go/testUnionsKinded_test.go

```diff
@@ -35,36 +31,46 @@ func TestUnionKinded(t *testing.T) {
 		}),
 	))
 
+	// These are the same *type-level* as in TestUnionKeyedComplexChildren,
+	//  but (of course) have very different representations.
```

Comment:
I've been entertaining the thought of putting some methods on this `testcase` structure so that one can "derive" variations of them with only some fields changed, which could be useful for tests like this that poke at variations in representation.

Could be a future work thing.